core/src/main/java/org/elasticsearch/monitor/jvm/HotThreads.java

```diff
@@ -294,10 +289,12 @@ public int compare(MyThreadInfo o1, MyThreadInfo o2) {
                 }
             }
             return sb.toString();
-        } finally {
-            if (enabledCpu) {
-                threadBean.setThreadCpuTimeEnabled(false);
-            }
+        } catch (Exception e) {
+            logger.warn("unable to capture hot_threads", e);
+            // Return as much as we were able to generate at least, plus the exception
+            sb.append("unable to capture hot_threads%n");
+            sb.append(ExceptionsHelper.stackTrace(e));
+            return sb.toString();
         }
```

Comment:
We don't strictly have to, however, if the exception propagates then the node is "skipped" in the output, I think the API is better served returning an exception for the node rather than silently failing. (especially if it were a mixed cluster where some nodes supported hot_threads and some do not)